src/overlays/actors/ovl_En_Skjneedle/z_en_skjneedle.c

```diff
@@ -14,8 +14,10 @@ void EnSkjneedle_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnSkjneedle_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnSkjneedle_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnSkjneedle_Draw(Actor* thisx, GlobalContext* globalCtx);
+s32 func_80B01F6C(EnSkjneedle* this);
```

Comment:
we generally like to keep a line break between the 4 main functions and the actor specific functions